updater/cmd/updater/main.go

```diff
@@ -106,9 +108,26 @@ func main() {
 		}
 
 		rb.Rollback()
-		logger.Fatal("Rollback complete")
+		removeTmpDir(logger, installDir)
+
+		logger.Error("Rollback complete")
+		fail(logger, installDir)
```

Comment:
This seems redundant. Calling `removeTmpDir` then call `fail` which calls `removeTmpDir`

Reply:
Good catch, this is redundant, removed the redundant calls.